modelci/finetuner/computer_vision_fine_tuning.py

```diff
@@ -360,39 +242,76 @@ def main(args: argparse.Namespace) -> None:
         to a temporary directory.
     """
 
-    with TemporaryDirectory(dir=args.root_data_path) as tmp_dir:
-
-        model = TransferLearningModel(dl_path=tmp_dir, **vars(args))
-
-        trainer = pl.Trainer(
-            weights_summary=None,
-            progress_bar_refresh_rate=1,
-            num_sanity_val_steps=0,
-            gpus=args.gpus,
-            min_epochs=args.nb_epochs,
-            max_epochs=args.nb_epochs,
+    with TemporaryDirectory(dir=args.root_data_path):
+        net = torch.hub.load('pytorch/vision:v0.6.0', args.backbone, pretrained=True)
+        freeze(module=net, train_bn=True)
+
+        num_ftrs = net.fc.in_features
```

Comment:
leave a todo and think of this very carefully.

line 249, 250 should be incorporated into our system and connected to the frontend

Reply:
leave to do for editor